packages/application-generic/src/health/standard-queue.health-indicator.ts

```diff
@@ -33,4 +33,22 @@ export class StandardQueueServiceHealthIndicator extends HealthIndicator {
       this.getStatus(this.INDICATOR_KEY, false)
     );
   }
+
+  async isWorkerHealthy(): Promise<HealthIndicatorResult> {
```

Comment:
I have concern with the name. This checks the queue situation, not the worker. So for me the name is misleading. Not sure if provides more value than the `isHealthy` method. Besides that the duplication logs the same without making any difference so it could be misleading.

Reply:
I see what you mean, what do you think about "is active"? do you have any other suggestions?
@p-fernandez